Automattic/jetpack
# Jetpack > Recommendations: Add Jetpack Golden Token support

This PR aims to add the new Jetpack Golden Token plan in the Jetpack recommendations (which is part of the "old" Jetpack dashboard).

Fixes #29674 

## Proposed changes:


* Add new Jetpack Golden Token assets
* Register new welcome page (including recommendation specific product description)
* Register new onboarding flow

### Other information:


## Jetpack product discussion



* p1HpG7-lga-p2
* p1HpG7-ljt-p2
* p1HpG7-lpL-p2

## Does this pull request change what data or activity we track or use?



We introduce a new product in the suggestions flow that will appear in the tracks, but the events themselves, etc., are untouched.

## Testing instructions:






* Spin up a site using this branch (either locally or JN)
* Connect the site to your WPCOM user account
* Use the jetpack/golden-token MC tool to give the site a "Golden Token" (2f939-pb)
  * You will have to follow the test instructions in D105968-code if it hasn't been merged yet since it - at the time of writing these instructions - issues a Jetpack Complete plan instead.
* Go to `/wp-admin/admin.php?page=jetpack#/recommendations/`
  * Notice that you need a fresh site or Jetpack options for the recommendations flow to be triggered (read: you should now have gone through the steps before on the site).
  * You can work around the above limitation by running the following commands to reset the old state: `wp jetpack options delete recommendations_data` and `wp jetpack options delete recommendations_step`
* Verify that the Jetpack Golden Token flow appears and looks like the below screenshot

## Screenshot(s)

### 1. step: Our new Jetpack Golden Token page

![Screenshot 2023-03-27 at 19 16 26](https://user-images.githubusercontent.com/3846700/228021009-9cbbf7e6-7869-470c-83ee-667134851b1c.png)

### 2. step: Backup (untouched - 100% reuse of existing component)

![Screenshot 2023-03-27 at 19 12 20](https://user-images.githubusercontent.com/3846700/228021103-5c44f104-01a5-4907-9ba9-6c4bbe59159f.png)

### 3. step: Scan (untouched - 100% reuse of existing component)

![Screenshot 2023-03-27 at 19 12 33](https://user-images.githubusercontent.com/3846700/228021250-13577a76-880b-4c32-a313-4f8dedc860d4.png)

### 4. step: Summary (untouched - 100% reuse of existing component)

There is two caveat on this page:

1. We say: `Part of your Jetpack Golden Token plan`
2. We show the existing Backup and Scan features - which means we promise real-time features for both.

![Screenshot 2023-03-27 at 19 17 44](https://user-images.githubusercontent.com/3846700/228021303-f9e0a5b2-8615-4c30-b5e8-469660633b3b.png)
